Azure/sonic-utilities
# [config] Add vlan member only after checking RIF is not present

#### What I did
Resolves https://github.com/Azure/sonic-buildimage/issues/7028
Updated mechanism how to check is the port has RIF or not before adding it the port to the Vlan.
The related unit tests updated.

Copied implementation of port_util from swsssdk to utilities_common of sonic_utilities. Replaced using of  swsssdk to utilities_common.

#### How I did it
By using functionality of `port_util.py` to check whether a RIF is present for the port in the system.
Now, the user can add a member to the Vlan only if there are no router records(`ASIC_STATE:SAI_OBJECT_TYPE_ROUTER_INTERFACE:`) in ASIC_DB, linked to OID of the interface.

#### How to verify it
1. sudo config interface ip add Ethernet0 30.0.0.1/24
2. sudo config route add prefix 20.0.0.1/32 nexthop Ethernet0
3. sudo config interface ip remove Ethernet0 30.0.0.1/24
4. sudo config vlan add 30
5. sudo config vlan member add --untagged 30 Ethernet0
  The last command will return the next error: `Error: Ethernet0 is a router interface! Check assigned L3 addresses, a related static routes etc.`
6. sudo config route del prefix 20.0.0.1/32 nexthop Ethernet0
This will delete the route and RIF for the port.
7. sudo config vlan member add --untagged 30 Ethernet0
The port will be added successfully, 

#### Previous command output
```
admin@sonic:~$ sudo config vlan member add 30 Ethernet0
Usage: config vlan member add [OPTIONS] <vid> port
Try "config vlan member add -h" for help.

Error: Ethernet0 is a router interface!
```

#### New command output
Output of command `config vlan member add` for the case is little bit clarified to help user to understand that is going on.
```
admin@sonic:~$ sudo config vlan member add 30 Ethernet0
Usage: config vlan member add [OPTIONS] <vid> port
Try "config vlan member add -h" for help.

Error: Ethernet0 is a router interface! Check assigned L3 addresses, a related static routes etc.
```
